test/test_prims.py

```diff
@@ -1124,6 +1124,28 @@ def test_philox_rand(self, device, dtype):
             for a, b in zip(references, results):
                 self.assertEqual(a, b)
 
+
+    @dtypes(torch.float32)
+    def test_functional_rng_wrappers(self, device, dtype):
+
+        torch.manual_seed(123)
+        ref1 = torch.rand(10, device=device, dtype=dtype)
+        ref2 = torch.rand(10, device=device, dtype=dtype)
+
+
+        torch.manual_seed(123)
+        rng_state1, res1 = torch._prims.rng_prims.run_and_save_rng_state(torch.rand, 10, device=device, dtype=dtype)
+        rng_state2, res2 = torch._prims.rng_prims.run_and_save_rng_state(torch.rand, 10, device=device, dtype=dtype)
+
+        torch.manual_seed(123)
```

Comment:
Shouldn't this not be needed?

Reply:
Ah. Good catch! yes, this is not needed. Fixing.